### PR TITLE
Add a cypress integration test for the linelist table

### PR DESCRIPTION
**Pull request by allysonjp715**
This definitely needs a lot of refinement, but achieves putting the cypress infrastructure in place.

Right now you must run the stack locally first, and then run the tests against the local server. In a future PR I will look to add this to our CI tests where it starts its own server before running. But for local development it's much easier to run your own local stack than to have the test start one up.

I left some example code + comments in while everyone is getting acquainted with Cypress.

### CODE REVIEW

**Comment by allysonjp715** [comment]
[Integration testing strategy](https://trello.com/c/dCCp9gpE/134-integration-testing-strategy)

File: verification/curator-service/ui/cypress/fixtures/example.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (khmoran) [inline verification/curator-service/ui/cypress/fixtures/example.json:1]: do we need to commit this file?

FWM if this was generated by a command and we'll clean up example stuff later

Comment (allysonjp715) [reply verification/curator-service/ui/cypress/fixtures/example.json:1]: Yes, twas "generated by a command and we'll clean up example stuff later"

File: verification/curator-service/ui/cypress/integration/components/LinelistTableTest.spec.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+describe('Linelist table', function () {
```
Comment (attwad) [inline verification/curator-service/ui/cypress/integration/components/LinelistTableTest.spec.js:1]: cypress apparently ships with typescript support, can we stick to typescript and avoid javascript please?

https://docs.cypress.io/guides/tooling/typescript-support.html#Install-TypeScript

Comment (allysonjp715) [reply verification/curator-service/ui/cypress/integration/components/LinelistTableTest.spec.js:1]: Ah, thanks for catching, just an oversight. Fixed in https://github.com/open-covid-data/healthmap-gdo-temp/pull/158

**Pull request merged**